packages/netlify-cms-proxy-server/webpack.config.js

```diff
@@ -25,6 +25,11 @@ module.exports = {
         test: /\.ts$/,
         use: ['ts-loader'],
       },
+      {
```

Comment:
Is this required? The proxy server is a node program and should not need to handle `css`

Reply:
No, not needed. Had some trouble running on Windows, but the branches I was working on were about ~400 commits behind. Should have backed this out when I rebased on master.

Back on my Mac, hopefully fine on Windows now the branch isn't so far behind.